collection-loader.js

```diff
@@ -1,28 +1,34 @@
-const get = require('lodash/get');
-const flatMap = require('lodash/flatMap');
+const get = require("lodash/get");
+const flatMap = require("lodash/flatMap");
 
 function loadCollectionItems(
   client,
   collections,
-  {storyFields, storyLimits, defaultNestedLimit}
+  { storyFields, storyLimits, defaultNestedLimit }
 ) {
   const bulkRequestBody = collections.reduce((acc, collection) => {
-    const limit =
-      storyLimits[get(collection, ['associated-metadata', 'layout'])] ||
-      defaultNestedLimit;
+    let limit = storyLimits[get(collection, ["associated-metadata", "layout"])];
+
+    if (!limit && get(collection, ["childCollectionLimit"])) {
```

Comment:
Why cant we use the storyLimits itself for this? Every collection has a storyLimit can we use the same number to specify the limits?

Reply:
As far as I see this the storyLimits map and the nestedLimits map is going to look pretty similar?

Reply:
The other way we can do this is the row has a collectionLimit, we should attach a story limit when we are building that row. Right now we need to manually compute it for every collection of collection rows which is not intuitive because they need to build the rows in one place and add the limits in another place.